package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "temporal_tables",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
don't bump the version, this is now automated

Reply:
Thanks, reverted.